app/src/main/java/com/github/libretube/services/DownloadService.kt

```diff
@@ -74,19 +75,21 @@ class DownloadService : Service() {
             )
             if (downloadType in listOf(DownloadType.VIDEO, DownloadType.AUDIO_VIDEO)) {
                 videoDownloadId = downloadManagerRequest(
-                    getString(R.string.video),
+                    videoName,
                     getString(R.string.downloading),
                     videoUrl,
+                    false,
```

Comment:
Now, when only downloading the audio stream of a video, there simply won't be any notification :wink: